docs/rfcs/0000-namespace.md

```diff
@@ -0,0 +1,76 @@
+- Start Date: (2015-10-29)
+- RFC PR: (leave this empty)
+- AMS Issue: (leave this empty)
+
+# Summary
+
+Provide a consistent API for the user of the AMS.
+
+# Motivation
+
+The actual public API is defined under `ActiveModelSerializers`,
+`ActiveModel::Serializer` and `ActiveModel`.
```

Comment:
And I guess I wrote some more at https://github.com/rails-api/active_model_serializers/pull/1301/files#r42963185